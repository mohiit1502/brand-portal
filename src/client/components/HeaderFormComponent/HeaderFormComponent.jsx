import React from "react";
import PropTypes from "prop-types";
import "./HeaderFormComponent.component.scss";

const HeaderFormComponent = props => {
  return (
    <div className={`c-HeaderFormComponent row form-prompt${props.containerClasses ? " " + props.containerClasses : ""}`}>
      <div className={`col${props.colClasses ? " " + props.colClasses : ""}`}>
        <p>{props.header}</p>
      </div>
    </div>
  );
};

HeaderFormComponent.propTypes = {
  containerClasses: PropTypes.string,
  header: PropTypes.string
};

export default HeaderFormComponent;
