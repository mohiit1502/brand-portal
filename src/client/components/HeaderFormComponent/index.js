import HeaderFormComponent from './HeaderFormComponent.jsx';

export default HeaderFormComponent;