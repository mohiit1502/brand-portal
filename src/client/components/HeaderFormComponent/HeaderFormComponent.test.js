/* eslint-disable filenames/match-regex, no-unused-vars, no-undef */
import React from "react";
import HeaderFormComponent from "./HeaderFormComponent";

describe("HeaderFormComponent", () => {
   it("renders without error", () => {

   });
});
