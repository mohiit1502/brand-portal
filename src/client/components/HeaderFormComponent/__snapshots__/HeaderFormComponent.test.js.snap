// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderFormComponent renders without error 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <MockNextContext
    pathname="/dashboard"
  >
    <MemoryRouter
      initialEntries={
        Array [
          "/dashboard",
        ]
      }
    >
      <Router
        history={
          Object {
            "action": "POP",
            "block": [Function],
            "canGo": [Function],
            "createHref": [Function],
            "entries": Array [
              Object {
                "hash": "",
                "pathname": "/dashboard",
                "search": "",
                "state": undefined,
              },
            ],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "index": 0,
            "length": 1,
            "listen": [Function],
            "location": Object {
              "hash": "",
              "pathname": "/dashboard",
              "search": "",
              "state": undefined,
            },
            "push": [Function],
            "replace": [Function],
          }
        }
      >
        <HeaderFormComponent
          colClasses=""
          containerClasses=""
          header=""
        >
          <div
            className="c-HeaderFormComponent row form-prompt"
          >
            <div
              className="col"
            >
              <p />
            </div>
          </div>
        </HeaderFormComponent>
         
      </Router>
    </MemoryRouter>
  </MockNextContext>
</Provider>
`;
